years_to_hours.rb

```diff
@@ -9,5 +9,12 @@
 puts "Enter a number of decades"  
 decades = gets.chomp # this returns a string  
 decades = decades.to_i # this converts a string to an integer  
-minutes = decades * 10 * 24 * 60 
-puts "That's #{minutes} minutes."
\ No newline at end of file
+minutes = decades * 10 * 365 * 24 * 60 
+puts "That's #{minutes} minutes."
+
+# part 3
+puts "Enter your age"  
+age = gets.chomp # this returns a string  
+age = age.to_i # this converts a string to an integer  
```

Comment:
Adding comments is a good practice, however, what this commented code does is very self explanatory and (in my personal opinion), I don't consider necessary adding comments here.

Usually comments are handy when we try to help ourselves when making code, when we have to document functions, or when we have code that is complex or hard to understand.

Nothing wrong with this code, it is very good, I'm just giving my 2 cents 👍 .